terraform/20-app/vars.tf

```diff
@@ -31,3 +31,9 @@ variable "single_nat_gateway" {
 }
 
 variable "halo_account_type" {}
+
+variable "api_gateway_stage_name" {
```

Comment:
Presumably once we get a prod account, we'll need to inject this variable in as `live`?

Reply:
Yes, I was planning to add `API_GATEWAY_STAGE_NAME: "live"` to `production.yml` in ticket CDD-2442